app/controllers/api/students_controller.rb

```diff
@@ -10,20 +10,20 @@ def create
   end
 
   def index
-    students = Student.includes(:school).page params[:page]
+    students = Student.includes(:group, :school).page params[:page]
     render json: students,
                  serializer: PaginatedSerializer,
                  each_serializer: StudentIndexSerializer
   end
 
   def show
-    student = Student.find params[:id]
+    student = Student.includes(comments: :user).find params[:id]
```

Comment:
This `.includes(comments: :user)` includes associated `comments` and then associated `user` of each comment.